ironic/drivers/modules/drac/raid.py

```diff
@@ -655,7 +693,7 @@ def _assign_disks_to_volume(logical_disks, physical_disks_by_type,
         # filter out disks without free disk space
         disks = [disk for disk in disks if free_space_mb[disk] > 0]
 
-        # sort disks by free size which is important if we have max disks limit
+        # sort disks by free size which is important if we  have max disks limit
```

Comment:
Can you undo the above change since it's not directly related to this patch?

Reply:
Thanks. removed single space.